### ISSUE TITLE
[Bug Report] el-select with multiple and filterable property can not be closed by clicking the input area

### ISSUE THREAD

**Issue by Annushkash** [ElemeFE/element#18098]
### Element UI version
0.1.0

### OS/Browsers version
75.0.3770.100

### Vue version
2.6.10

### Reproduction Link
https://codepen.io/AnnushkaSh/pen/aggoPE

### Steps to reproduce
Create **multiple and filterable** el-select.
Click on select component and then click on the input area.

### What is Expected?
el-select-dropdown component will be closed

### What is actually happening?
el-select-dropdown component is not closing

It's dublicate #16559 and fixed in current version (2.12), but be reproduced again :-( .

**Comment by hjfrun** [comment]
Yes, I also encountered the same problem. The real source of the issue is the `filterable` attribute. This is really an annoying bug.

**Comment by Annushkash** [comment]
> Yes, I also encountered the same problem. The real source of the issue is the `filterable` attribute. This is really an annoying bug. My version is `2.12.0`, the latest version currently.

I'm sorry, I use current version too, edit my ussue.

**Comment by Emiya0306** [comment]
I don't think it should be close easily.
We can consider the case which follow these steps below.
1. I input the search string (like "appe", actually should be "apple") in multiple and filterable el-select.
1. I will click the string between "app" and "e", the dropdown will be closed! I have no way to edit.

**Comment by Garine519** [comment]
any solution for this?
